py/server/deephaven/updateby.py

```diff
@@ -580,21 +716,27 @@ def cum_count_where(col: str, filters: Union[str, Filter, List[str], List[Filter
         DHError
     """
     if not isinstance(col, str):
-        raise DHError(message="count_where aggregation requires a string value for the 'col' argument.")
-    filters = to_sequence(filters)
+        raise DHError(
+            message="count_where aggregation requires a string value for the 'col' argument."
+        )
+    filters = to_sequence(filters)  # type: ignore[arg-type]
 
     try:
-        return UpdateByOperation(j_updateby_op=_JUpdateByOperation.CumCountWhere(col, and_(filters).j_filter))
+        return UpdateByOperation(
+            j_updateby_op=_JUpdateByOperation.CumCountWhere(col, and_(filters).j_filter)
+        )
     except Exception as e:
-        raise DHError(e, "failed to create a cumulative count_where UpdateByOperation.") from e
+        raise DHError(
+            e, "failed to create a cumulative count_where UpdateByOperation."
+        ) from e
 
 
-def forward_fill(cols: Union[str, List[str]]) -> UpdateByOperation:
+def forward_fill(cols: Union[str, Sequence[str]]) -> UpdateByOperation:
     """Creates a forward fill UpdateByOperation for the supplied column names. Null values in the columns are
     replaced by the last known non-null values. This operation is forward only.
 
     Args:
-        cols (Union[str, List[str]]): the column(s) to be operated on, can include expressions to rename the output,
+        cols (Union[str, list[str]]): the column(s) to be operated on, can include expressions to rename the output,
```

Comment:
Notice that the type is wrong.  I think I've seen this in a number of other functions in this file.